### PR TITLE
Update CI badge in README.md.

### PR DESCRIPTION
**Pull request by simi**
- check GitHub actions status instead of previously used Travis CI

### CODE REVIEW

**Comment by simi** [comment]
@pointlessone I think travis-ci should be removed from "required" checks now, since it is invalid. Once done I can force push again to retrigger CI checks and :pray: make this green.

**Comment by pointlessone** [comment]
I believe, I have removed last traces of Travis. Try again?

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Prawn: Fast, Nimble PDF Generation For Ruby
 
 [![Gem Version](https://badge.fury.io/rb/prawn.svg)](http://badge.fury.io/rb/prawn)
-[![Build Status](https://api.travis-ci.org/prawnpdf/prawn.svg?branch=master)](http://travis-ci.org/prawnpdf/prawn)
+[![Build Status](https://github.com/prawnpdf/prawn/workflows/CI/badge.svg)](https://github.com/prawnpdf/prawn/actions?query=branch%3Amaster)
```
Comment (pointlessone) [inline README.md:4]: I wonder if there's a way to not hardcode the branch name here? Is there a way to filter to _default_ branch?

Comment (donv) [reply README.md:4]: We can do this later, if there is a way.

**Review by donv — APPROVED** [review]

**Comment by donv** [comment]
I think this one is ready to be merged.

**Comment by pointlessone** [comment]
Thank you for your contribution.

**Pull request merged**